app/views/device/show.html.erb

```diff
@@ -1,79 +1,191 @@
-<h1><%= "Info for device #{@device.display_name}" %> </h1>
+<h1 class="mb-3">
+  <%= @device.display_name %>
+  <span
+    class="flag server-page fi fi-<%= @device.two_digit_location.downcase %> fis"
+    title="<%= @device.display_name %> is located in <%= @device.full_country_name %>">
+  </span>
+</h1>
+<p class="big-text mb-5">Entered by: <%= @device.pretty_owner %></p>
 
-<style>
-table, th, td {
-  border:1px solid black;
-}
-td {
-  text-align: center;
-}
-</style>
+<div class="cl-card flex-column-centred">
+  <div class="device-type-wrapper">
+    <%= image_tag "#{@device.platform_icon}.png" %>
+    <div>
+      <h4><%= @device.platform.capitalize %></h4>
```

Comment:
One side-effect of using `.capitalize` here is that it unfortunately doesn't know that `aws` is an acronym and converts it into `Aws` instead of `AWS`.

There are a few ways to solve this, but IMO the best solution would be:
* Change `.capitalize` to `.titleize`
* Uncomment/set the following lines in `/config/initializers/inflections.rb`:
```
ActiveSupport::Inflector.inflections(:en) do |inflect|
  inflect.acronym 'AWS'
end
```
since it allows us to re-use this capitalization method on platforms elsewhere if we need to.

Another issue with this line is that it doesn't distinguish between Alces Cloud and regular OpenStack instances. The platform logo image is correctly updated, but for Alces Cloud instances it shows:
![image](https://github.com/openflighthpc/carbon-leaderboard/assets/57297493/7b542cc1-e35b-4efa-b398-8f743d7ff3b8)

I think this should say `Alces Cloud` instead.